app/models/user/notifications/mentor_finished_discussion_notification.rb

```diff
@@ -0,0 +1,24 @@
+class User
+  module Notifications
+    class MentorFinishedDiscussionNotification < Notification
+      params :discussion
+
+      def url = discussion.student_url
+      def image_type = :avatar
+      def image_url = mentor.avatar_url
+      def guard_params = "Discussion##{discussion.id}"
```

Comment:
As a test, I've used Ruby 3 style single-line methods. To me, that looks great, but I don't know how you feel about it?

Note: if you'd prefer not to, I'll revert

Reply:
Yep - they're great.